### ISSUE TITLE
[Task]: Display mock data in a list using USWDS layout

### ISSUE THREAD

**Issue by sumiat** [HHS/simpler-grants-gov#1296]
### Summary

Mock data without pagination, just a long list of opportunities

-May have already done the work of displaying mock data as part of this ticket => #1294 

### Acceptance criteria

- Mock data displaying only the opportunity list
- 2 columns => 1 for filter (which will just be blank to start) , and another for the search results

Blocked by #1282 

![image](https://github.com/HHS/simpler-grants-gov/assets/93001277/4b461be8-cf96-486e-bbad-c2f8e3baebbd)

**Comment by rylew1** [comment]
cc @andycochran @crystabelrangel  - let me know if there's anything in particular about using the USWDS layout or this first  layout pass that we want to see. Just trying to keep it simple and show the search bar expanding across all the columns, 1 column for the filters (which will just be blank to start), and another column for the actual search results.


It looks like the layout grid is a 12 column system? Are there any rules on how many columns the filter area and search results area?

**Comment by andycochran** [comment]
Let's try `tablet:grid-col-4` and `tablet:grid-col-8`

**Comment by rylew1** [comment]
On hold until we sort out the app router upgrade work
